feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -176,51 +174,64 @@ func configForwardingPolicy() *oc.NetworkInstance_PolicyForwarding {
+	fwdPolicy3.SetType(oc.Policy_Type_PBR_POLICY)
 	fwdPolicy3.GetOrCreateRule(1).GetOrCreateIpv4().DestinationAddress = ygot.String(ipv4Address)
 	fwdPolicy3.GetOrCreateRule(1).GetOrCreateAction().NetworkInstance = ygot.String("10")
-	fwdPolicy3.GetOrCreateRule(2).GetOrCreateIpv6().DestinationAddress = ygot.String(ipv6Address)
-	fwdPolicy3.GetOrCreateRule(2).GetOrCreateAction().NetworkInstance = ygot.String("20")
+	fwdPolicy3.GetOrCreateRule(2).GetOrCreateIpv6().Protocol = oc.UnionUint8(icmp6)
+	fwdPolicy3.GetOrCreateRule(2).GetOrCreateAction().Discard = ygot.Bool(false)
+	fwdPolicy3.GetOrCreateRule(3).GetOrCreateIpv6().DestinationAddress = ygot.String(ipv6Address)
+	fwdPolicy3.GetOrCreateRule(3).GetOrCreateAction().NetworkInstance = ygot.String("20")
 
 	fwdPolicy4 := policyFwding.GetOrCreatePolicy("match-ipip-dscp46")
+	fwdPolicy4.SetType(oc.Policy_Type_PBR_POLICY)
 	fwdPolicy4.GetOrCreateRule(1).GetOrCreateIpv4().Protocol = oc.UnionUint8(ipipProtocol)
 	fwdPolicy4.GetOrCreateRule(1).GetOrCreateIpv4().Dscp = ygot.Uint8(46)
 	fwdPolicy4.GetOrCreateRule(1).GetOrCreateAction().NetworkInstance = ygot.String("10")
-	fwdPolicy4.GetOrCreateRule(2).GetOrCreateIpv4().DestinationAddress = ygot.String(ipv4Address)
-	fwdPolicy4.GetOrCreateRule(2).GetOrCreateIpv6().DestinationAddress = ygot.String(ipv6Address)
-	fwdPolicy4.GetOrCreateRule(2).GetOrCreateAction().Discard = ygot.Bool(true)
+	fwdPolicy4.GetOrCreateRule(2).GetOrCreateIpv6().Protocol = oc.UnionUint8(icmp6)
+	fwdPolicy4.GetOrCreateRule(2).GetOrCreateAction().Discard = ygot.Bool(false)
+	fwdPolicy4.GetOrCreateRule(3).GetOrCreateIpv4().DestinationAddress = ygot.String(ipv4Address)
+	fwdPolicy4.GetOrCreateRule(3).GetOrCreateIpv6().DestinationAddress = ygot.String(ipv6Address)
+	fwdPolicy4.GetOrCreateRule(3).GetOrCreateAction().Discard = ygot.Bool(true)
 
 	fwdPolicy5 := policyFwding.GetOrCreatePolicy("match-ipip-dscp42or46")
+	fwdPolicy5.SetType(oc.Policy_Type_PBR_POLICY)
 	fwdPolicy5.GetOrCreateRule(1).GetOrCreateIpv4().Protocol = oc.UnionUint8(ipipProtocol)
 	fwdPolicy5.GetOrCreateRule(1).GetOrCreateIpv4().Dscp = ygot.Uint8(42)
 	fwdPolicy5.GetOrCreateRule(1).GetOrCreateAction().NetworkInstance = ygot.String("10")
 	fwdPolicy5.GetOrCreateRule(2).GetOrCreateIpv4().Protocol = oc.UnionUint8(ipipProtocol)
 	fwdPolicy5.GetOrCreateRule(2).GetOrCreateIpv4().Dscp = ygot.Uint8(46)
 	fwdPolicy5.GetOrCreateRule(2).GetOrCreateAction().NetworkInstance = ygot.String("10")
-	fwdPolicy5.GetOrCreateRule(3).GetOrCreateIpv4().DestinationAddress = ygot.String(ipv4Address)
-	fwdPolicy5.GetOrCreateRule(3).GetOrCreateIpv6().DestinationAddress = ygot.String(ipv6Address)
-	fwdPolicy3.GetOrCreateRule(3).GetOrCreateAction().Discard = ygot.Bool(true)
+	fwdPolicy5.GetOrCreateRule(3).GetOrCreateIpv6().Protocol = oc.UnionUint8(icmp6)
+	fwdPolicy5.GetOrCreateRule(3).GetOrCreateAction().Discard = ygot.Bool(false)
+	fwdPolicy5.GetOrCreateRule(4).GetOrCreateIpv4().DestinationAddress = ygot.String(ipv4Address)
```

Comment:
match on destination not required as per HLD

Reply:
any IPv6 address is destination here to allow IPv6 to NI 20, same like any IPv4 packet to NI 10.